src/template/template.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { readFileSync } from 'fs';
+import { Content } from './dto/content.dto'
+import * as Handlebars from 'handlebars';
+import mjml2html = require('mjml');
+
+@Injectable()
+export class TemplateService{
+
+  getResetPasswordEmail<T>(data: T): string {
+    const template = this.getResetPasswordEmailTemplate("en");
+    return this._injectData(template, data);
+  }
+
+  public getResetPasswordEmailTemplate(locale: string): Content {
+    let content = new Content();
+    content.mjml = readFileSync(`${__dirname}/templates/${locale}/reset-password.mjml`, 'utf8');
+    return content;
+  }
+
+  private _injectData<T>(content: Content, data: T): string {
```

Comment:
leave underscore for consistency [prev. code base example](https://github.com/BrewInteractive/authentication-service-nestjs/blob/02530933d4526ba5a233cc6b4e0bd1a99e18c11c/src/user/user.service.ts#L200)